var/spack/repos/builtin/packages/mpfr/package.py

```diff
@@ -46,7 +47,7 @@ class Mpfr(AutotoolsPackage, GNUMirrorPackage):
         description="Build shared libs, static libs or both",
     )
 
-    force_autoreconf = True
+    # force_autoreconf = True
```

Comment:
Why is this commented out (alternatively, why was this `True` in the first place)?

Reply:
I _think_ it was true because of an old config.guess or similar that wasn't playing nice with some architecture.  That doesn't seem to be the case now, and dropping it removes several build dependencies.  I'm not sure it actually matters anymore though, this dates from when I was trying to untangle the C++ bootstrap cycle between libstdc++, gcc and gmp that has since been solved better by building gmp without its c++ bindings in the second stage.  Will try it.